CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ x.x.x Release notes (yyyy-MM-dd)
   types.
 * Remove `RLMPropertyTypeArray` in favor of a separate bool `array` property on
   `RLMProperty`/`Property`.
+* Realm Swift APIs that filter or look up the index of an object based on a
```

Comment:
I don't feel strongly about getting this into a 2.x release, so feel free to leave this on `master-3.0`. My question was mainly about whether this change belongs in the "Breaking changes" section of the change log.